pkg/plugins/adapters/adapters.go

```diff
@@ -17,6 +17,7 @@ func ModelToInstanceSettings(ds *models.DataSource) (*backend.DataSourceInstance
 		ID:                      ds.Id,
 		Name:                    ds.Name,
 		URL:                     ds.Url,
+		UID:                     ds.Uid,
```

Comment:
^^

Reply:
I think this means we could now use the uid as the datasource channel path rather than the id!

Reply:
Yep, will try to do in separate pr after merging this